dune

```diff
@@ -0,0 +1 @@
+(data_only_dirs cohttp-lwt-jsoo cohttp_lwt_jsoo_test)
```

Comment:
We don't need this dune file.

Reply:
just so that dune doesn't build jsoo packages.

Reply:
I see. Note that you could always build just the package you want with `dune build cohttp-eio.install`.